helpers/api.js

```diff
@@ -36,10 +40,10 @@ const request = (method, url, bodyParams, queryParams) => {
   return axios(options);
 };
 
-const get = (url, queryParams) => request('get', url, null, queryParams);
-const post = (url, bodyParams, queryParams) => request('post', url, bodyParams, queryParams);
-const put = (url, bodyParams, queryParams) => request('put', url, bodyParams, queryParams);
-const patch = (url, bodyParams, queryParams) => request('patch', url, bodyParams, queryParams);
+const get = (url, queryParams) => request('get', url, null, queryParams, {});
+const post = (url, bodyParams, queryParams, queryHeaders) => request('post', url, bodyParams, queryParams, queryHeaders);
+const put = (url, bodyParams, queryParams) => request('put', url, bodyParams, queryParams, {});
+const patch = (url, bodyParams, queryParams) => request('patch', url, bodyParams, queryParams, {});
```

Comment:
Remove all empty object `{}` params. Rename queryHeaders to just `headers` please.